docs/DeveloperGuide.md

```diff
@@ -226,23 +226,28 @@ This feature is used to find stalls and reviews in FoodWhere by name and/or by t
 * `sfind`: Finds stalls
 * `rfind`: Finds reviews
 
-`sfind` allows users to find stalls in `AddressBook` by names, through matching of input keyword(s) with stall names. Additionally, users can find stalls by tags, through matching of input keyword(s) with stall tags.
+`sfind` allows users to find stalls in `AddressBook` by `name`, through matching of input keyword(s) with stall names. Additionally, users can find stalls by `tags`, through matching of input keyword(s) with stall tags.
```

Comment:
I'm not sure why tags and names are now in backticks, are these properties on the Stall class?

Reply:
Meant to be ** tags ** instead, thanks for the catch